### PR TITLE
edencommon fb303 fbthrift fizz folly mvfst proxygen wangle watchman 2024.11.25.00

### PR DESCRIPTION
**Pull request by chenrui333**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by carlocab** [comment]
Failure looks like https://github.com/GuillaumeGomez/sysinfo/issues/1392

**Comment by chenrui333** [comment]
yeah, the issue got fixed with sysinfo 0.32.1, but bumping the dependency cause other build failures.

File: Formula/w/watchman.rb [hunk 1]
```diff
@@ -71,3 +74,16 @@ def install
...
+
+__END__
+--- a/watchman/cli/Cargo.toml
++++ b/watchman/cli/Cargo.toml
+@@ -16,7 +16,7 @@
+ serde = { version = "1.0.185", features = ["derive", "rc"] }
+ serde_json = { version = "1.0.132", features = ["float_roundtrip", "unbounded_depth"] }
+ structopt = "0.3.26"
+-sysinfo = "0.30.11"
++sysinfo = "0.32.1"
```
Comment (chenrui333) [inline Formula/w/watchman.rb:86]: somehow that did not work in my local :(

**Review by branchv — APPROVED** [review]

**Review by daeho-ro — APPROVED** [review]

**Comment by carlocab** [comment]
@chenrui333, can you upstream the patch?

**Comment by chenrui333** [comment]
> @chenrui333, can you upstream the patch?

yeah, totally. 👍

**Pull request merged**